README.md

```diff
@@ -1,5 +1,82 @@
 # PayPal iOS SDK
 
-![GitHub Actions Tests](https://github.com/paypal/iOS-SDK/workflows/Tests/badge.svg)
+Welcome to PayPal's iOS SDK. This library will help you accept card, PayPal, Venmo, and alternative payments in your iOS app.
 
-[WIP] One merchant integration point for all of PayPal's services
+## Support
+
+The PayPal iOS SDK supports a minimum deployment target of iOS 13+ and requires Xcode 13+. See our [Client Deprecation policy](https://developer.paypal.com/braintree/docs/guides/client-sdk/deprecation-policy/ios/v5) to plan for updates.
+
+### Package Managers
+This SDK supports:
+
+* CocoaPods
+* Swift Package Manager
+
+### Languages
+
+This SDK supports Swift 5.1+. This SDK is written in Swift.
+
+### UI Frameworks
+This SDK supports:
+
+* UIKit
+* SwiftUI
+
+## Modularity
+
+The PayPal iOS SDK is comprised of various submodules.
+* `Card`
+* `PayPal`
+* ...
+
+To accept a certain payment method in your app, you only need to include that payment-specific submodule.
+
+## Sample Code
+
+```
+// STEP 0: Fetch an ACCESS_TOKEN and ORDER_ID from your server.
+
+// STEP 1: Create a PaymentConfiguration object
+paymentConfig = PaymentConfig(token: ACCESS_TOKEN)
+
+// STEP 2: Create payment method client objects
+cardClient = CardClient(config: paymentConfig)
+
+// STEP 3: Collect relevant payment method details
+card = Card(number: 4111111111111111, cvv: 123, ...)
+
+// STEP 4: Call checkout method
+cardClient?.checkoutWithCard(orderID: ORDER_ID, card: card) { (result, error) in
+    if (error != nil) {
+        // handle checkout error
+        return
+    }
+    
+    guard let orderID = result?.orderID else { return }
+    // Send orderID to your server to process the payment
+}
```

Comment:
This snippet could be simplified if we use the `Result` type (even though this is just an example, i think this is clearer): 

```suggestion
cardClient?.checkoutWithCard(orderID: ORDER_ID, card: card) { result in
    switch result {
    case .success(let orderId):
      // Send orderID to your server to process the payment
    case .error(let error):
      // handle checkout error
    }
}
```

Reply:
Good call! Let's definitely use the Swift `Result` type where we can.